src/summary/history/notes-helper.ts

```diff
@@ -0,0 +1,24 @@
+import dayjs from "dayjs";
+
+export function getAuthorName(note: any): string {
+  let author: any;
+  if (note.participant) {
+    author = note.participant[0].individual.display;
+    return author.toUpperCase();
+  } else if (note.extension) {
+    author = note.extension.find(ext => ext.url === "changedBy");
+    return author ? author.valueString.toUpperCase() : "";
+  }
+}
+export function getNotes(notes) {
+  return notes.map(note => note.resource);
+}
+export function formatNotesDate(date: string): string {
```

Comment:
is this the same implementation as that of `formatDate` in dimension-helper?

Reply:
As per Figma designs date display format for notes is slightly different

Reply:
![datesdiff](https://user-images.githubusercontent.com/19890755/69872473-873d7d80-12c6-11ea-9d14-b00d51a007b9.png)
![vitalsheight](https://user-images.githubusercontent.com/19890755/69872474-87d61400-12c6-11ea-9e3d-93bb98877aa2.png)
this is the differents for the dimension the year starts when its less than this year